CRM/Usermenu/Page/UserMenu.php

```diff
@@ -33,10 +23,9 @@ class CRM_Usermenu_Page_UserMenu extends CRM_Core_Page {
   public function run() {
     $this->contactData();
 
+    $this->assign('menuItems', $this->getMenuItems());
     $this->assign('username', $this->getUserName());
     $this->assign('image', $this->getUserImagePath());
```

Comment:
Are we still pulling the user image from Civicrm or switching to Drupal avatars?

Reply:
Yes, it was confirmed that we should only use CiviCRM contact image.